packages/scaffold-config/src/frameworks.ts

```diff
@@ -17,11 +17,14 @@ export type CodeGenFramework = Cypress.ResolvedComponentFrameworkDefinition['cod
 export async function isDependencyInstalled (dependency: Cypress.CypressComponentDependency, projectPath: string): Promise<Cypress.DependencyToInstall> {
   try {
     debug('detecting %s in %s', dependency.package, projectPath)
-    const loc = require.resolve(path.join(dependency.package, 'package.json'), {
-      paths: [projectPath],
-    })
 
-    const pkg = await fs.readJson(loc) as PkgJson
+    const packageFilePath = resolvePackagePath(dependency.package, projectPath) || ''
```

Comment:
Maybe I'm missing something, but couldn't we do this a bit more simply?

```suggestion
    const packageFilePath = resolvePackagePath(dependency.package, projectPath)
    if (!packageFilePath) {
      throw Error('Unable to resolve dependency')
    }
```

Reply:
yeah good call